notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -309,6 +313,12 @@ internal class NotifiqueListView(
       timestamp.text = dateFormatter.format(Date(notifique.timestamp))
       title.text = notifique.title
       message.text = notifique.message
+      try {
+        appPicture.setImageDrawable(context.packageManager.getApplicationIcon(notifique.package_))
+      }
+      catch (e: PackageManager.NameNotFoundException) {
```

Comment:
style: move catch to same line i.e. `} catch ...`

Reply:
changed